keyboards/crkbd/keymaps/gotham/keymap.c

```diff
@@ -0,0 +1,140 @@
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                          _______, _______, _______,    _______,   LWDEL, _______\
+                                      //`--------------------------'  `--------------------------'
+  ),
+
+  [_RAISE] = LAYOUT_kc( \
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+          ESC,    HASH,     DLR,    LCBR,    RCBR, ___X___,                      ___X___,    AMPR,    PIPE,    EXLM,    ASTR, ___X___,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      _______,    EXLM,      AT,    LPRN,    RPRN, ___X___,                      ___X___,    UNDS,     EQL,    ANGL,    ANGR, ___X___,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      _______,    PERC,    CIRC,    LBRC,    RBRC, ___X___,                      ___X___,    MINS,    PLUS, ___X___,    BSLS, _______,\
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                          _______, _______, _______,    _______, _______, _______\
+                                      //`--------------------------'  `--------------------------'
+  ),
+
+  [_ADJUST] = LAYOUT_kc( \
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+      ___X___,  CK_RST, CK_DOWN,   CK_UP, CK_TOGG, RGB_TOG,                       MU_TOG,     F12,      F7,      F8,      F9, ___X___,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      ___X___, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, RGB_MOD,                       MU_MOD,     F11,      F4,      F5,      F6,   RESET,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      ___X___, RGB_HUD, RGB_SAD, RGB_VAD, RGB_SPD,  RGBRST,                       AU_TOG,     F10,      F1,      F2,      F3, _______,\
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                          _______, _______, _______,    _______,    VOLD,    VOLU\
+                                      //`--------------------------'  `--------------------------'
+  )
+};
+
+#if defined(RGBLIGHT_ENABLE) || defined(RGB_MATRIX_ENABLE)
+layer_state_t layer_state_set_user(layer_state_t state) {
+  /* For any layer other than default, save current RGB state and switch to layer-based RGB */
+  if (state == default_layer_state) {
```

Comment:
This can/would be better expressed as: 

```suggestion
    if (layer_state_cmp(state, 0)) {
```

As this would mean that it's on the default layer, and no other layers are active.

Reply:
Ah, I see. I'll fix these.

What's the issue with LAYOUT_kc, and what is recommended in its place? I'm still getting familiarized with the QMK code base, so pardon my ignorance.

Reply:
Done!